src/github/pullRequestManager.ts

```diff
@@ -771,6 +771,101 @@ export class PullRequestManager implements vscode.Disposable {
 		};
 	}
 
+	async getIssues(options: IPullRequestsPagingOptions = { fetchNextPage: false }): Promise<PullRequestsResponseResult> {
+		if (!this._githubRepositories || !this._githubRepositories.length) {
+			return {
+				pullRequests: [],
+				hasMorePages: false,
+				hasUnsearchedRepositories: false
+			};
+		}
+
+		const queryId = PRType.All.toString();
+		const getTotalFetchedPages = () => this.totalFetchedPages.get(queryId) || 0;
+		const setTotalFetchedPages = (numPages: number) => this.totalFetchedPages.set(queryId, numPages);
+
+		for (const repository of this._githubRepositories) {
+			const remoteId = repository.remote.url.toString() + queryId;
+			if (!this._repositoryPageInformation.get(remoteId)) {

```

Comment:
This map stores the number of pages we've fetched so far for PRs so that we can restore to the correct state when the tree refreshes. Right now it looks like this reuses the key for all PRs, so loading more PRs would cause issues to also get paged. I think we should use an issue-specific key in the map